content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
     caption="A service mesh after a control plane instance fails"
     >}}
 
-The following list ranks control plane deployment examples by availability:
+### Independent Control Planes
+
+You can also deploy a control plane per cluster, with a shared root CA. Each cluster's control plane
+manages only the workloads in its own cluster. If the control plane is unavailable (due to upgrade or
+temporary outage), the workloads will continue to function with the existing sidecar configuration and
+will re-attempt to connect to the local control plane periodically.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+The following list ranks control plane deployment examples by availability, configuration isolation:
+
+- One control plane per region (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per zone
+- Multiple control planes per zone
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority or the same trust
+domain depending on the following situations:
+
+- If you are using a vendor hosted CA that is shared across different customers,
+  then it is imperative to use a single trust domain for the entire mesh, as the
+  root CA across different customers could be the same (i.e. the vendor's CA itself).
+
+- If you have your own root CA for all your clusters, you can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
+  
+If you use a single shared trust domain, a service in a given namespace is globally unique - i.e. if
```

Comment:
This should talk specifically about service accounts or service identities, not just services.

Also we support aliasing so even if two clusters have the same service account foo in namespace ns but are in different trust domains, they _may_ be treated the same if the trust domains are aliased.

Reply:
Right, thats different trust domains. the sentence is about single trust domain